### PR TITLE
Add Kubernetes apis

### PR DESCRIPTION
**Pull request by alejandrojnm**
- all ready to integrate the Kubernetes part

### CODE REVIEW

File: kubernetes.go [hunk 1]
```diff
@@ -0,0 +1,208 @@
...
+	body, err := c.SendPostRequest(fmt.Sprintf("/v2/kubernetes/clusters/%s", id), v)
+	if err != nil {
+		return nil, err
+	}
+
+	return c.DecodeSimpleResponse(body)
+}
+
+// VersionKubernetesCluster returns all version of kubernetes in the cloud
+func (c *Client) VersionKubernetesCluster() ([]KubernetesVersions, error) {
```
Comment (andyjeffries) [inline kubernetes.go]: ListAvailableKubernetesVersions is a better name

File: kubernetes.go [hunk 2]
```diff
@@ -54,7 +54,7 @@ type Kubernetes struct {
 	InstalledApplications []KubernetesApplications `json:"installed_applications"`
 }
 
-type KubernetesConfig struct {
+type KubernetesClusterConfig struct {
 	Name              string `form:"name"`
 	NumTargetNodes    string `form:"num_target_nodes"`
```
Comment (andyjeffries) [inline kubernetes.go]: This should be an integer though, right?

**Comment by andyjeffries** [comment]
One comment left and other than that, ready for merging.

File: kubernetes.go (hunk 2 above)
Comment (alejandrojnm) [reply kubernetes.go]:
```suggestion
	NumTargetNodes    int `form:"num_target_nodes"`
```
i believe yes :D

**Comment by andyjeffries** [comment]
Thanks @alejandrojnm !

**Pull request merged**